CHANGELOG.md

```diff
@@ -3,7 +3,34 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/#semantic-versioning-200).
 
+## [1.0.2] - 2023-01-30
+### :magic_wand: Added
+- New plugin allowing users to override the driver name ([PR #371](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/371))
+- Default list of plugins added to the parameter 'wrapperPlugins' ([PR #332](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/332))
+- 2-node test cluster configuration ([PR #345](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/345))
+- Read-write splitting plugin example using Spring JDBC template. See example [here](./examples/AWSDriverExample/src/main/java/software/amazon/ReadWriteSplittingSpringJdbcTemplateExample.java)
```

Comment:
Can we visually group similar changes? E.g. put the r/w splitting changes near each other.